src/app.controller.ts

```diff
@@ -1,4 +1,5 @@
 import { Controller, Get } from '@nestjs/common';
+
```

Comment:
Maybe avoid adding this changes to the commit/PR. I saw there a few files with only this changes. WDYT?

Reply:
Totally agree, but this change is because of a lint rule which groups the imports, in the future this kind of changes I will separate this from a PR feature